prime-router/docs/design/proposals/translation-template-test-plan.md

```diff
@@ -0,0 +1,57 @@
+# Translation Template Test Plan
+
+## Problem Statement
```

Comment:
I'm wondering if we should get a little more specific here, something like "we wants tests to verify that we losslessly covert NIST 2.5.1 HL7 to FHIR and back to NIST 2.5.1 HL7", in other words we want a guarantee that if we get valid NIST 2.5.1 HL7 we can send it character for character the same to a receiver.